Index: GraduationProject/DockerFile
===================================================================
diff --git a/GraduationProject/DockerFile b/GraduationProject/DockerFile
--- a/GraduationProject/DockerFile	
+++ b/GraduationProject/DockerFile	
@@ -5,4 +5,4 @@
 FROM openjdk:17.0.1-jdk-slim
 COPY --from=build /target/Quizapp-0.0.1-SNAPSHOT.jar Quizapp.jar
 EXPOSE 8080
-ENTRYPOINT ["java","-jar","Quizapp.jar"]
\ No newline at end of file
+ENTRYPOINT ["java","-jar","Quizapp.jar"]
